AirCasting/CreateSessionViews/CreateSessionDetails/CreateSessionDetailsView.swift

```diff
@@ -0,0 +1,161 @@
+//  Created by Anna Olak on 24/02/2021.
+//
+import AirCastingStyling
+import SwiftUI
+
+struct CreateSessionDetailsView: View {
+    
+    @EnvironmentObject private var sessionContext: CreateSessionContext
+    @StateObject var viewModel: CreateSessionDetailsViewModel
+    @Binding var creatingSessionFlowContinues: Bool
+    
+    var body: some View {
+        GeometryReader { geometry in
+            ScrollView(.vertical) {
+                    VStack(alignment: .leading, spacing: 25) {
+                        ProgressView(value: 0.75)
+                        titleLabel
+                        VStack(spacing: 20) {
+                            sessionNameField
+                            sessionTagsField
+                        }
+                        if sessionContext.sessionType == SessionType.fixed {
+                            placementPicker
+                            transmissionTypePicker
+                            if viewModel.shouldShowCompleteCredentials() {
```

Comment:
after extracting the containing `if` body, do the same extraction here with the `if` and `else if`. 

And observe how the code becomes easily top-down readable without the need to focus on low-level internals ;)